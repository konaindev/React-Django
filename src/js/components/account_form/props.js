export const props = {
  steps: [
    { name: "Set Password", isComplete: true },
    { name: "Complete Account", isActive: true }
  ],
  title: "Complete your account",
  subtitle:
    "We need just a bit more information about you to complete your account."
};
