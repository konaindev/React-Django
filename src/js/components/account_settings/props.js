const user = {
  account_id: 1,
  account_name: "Remarkably",
  account_settings_url: "/users/account-settings",
  email: "test@psl.com",
  logout_url: "/users/logout/",
  user_id: "usr_jjzpeyfeshzpaha5"
};

const itemsOrder = ["profile", "lock", "email"];

const rules = [
  {
    label: "Be at least 8 characters",
    key: "length"
  },
  {
    label: "Contain alphabetic characters",
    key: "characters"
  },
  {
    label: "Not match personal information",
    key: "personal"
  },
  {
    label: "Not be a commonly used password",
    key: "used"
  }
];

const company_roles = [
  { label: "Owner", value: "owner" },
  { label: "Developer", value: "developer" },
  { label: "Asset Manager", value: "asset_manager" },
  { label: "Property Manager", value: "property_manager" }
];

const office_options = [
  { label: "Global", value: 1 },
  { label: "National", value: 2 },
  { label: "Regional", value: 3 },
  { label: "Other", value: 4 }
];

const office_countries = [
  { label: "United States of America", value: "USA" },
  { label: "United Kingdom", value: "GBR" }
];

const us_state_list = [
  { label: "Washington", value: "Washington" },
  { label: "New York", value: "New York" }
];

const gb_county_list = [
  { label: "London", value: "London" },
  { label: "Aberdeen", value: "Aberdeen" }
];

const profile = {
  avatar_url:
    "https://lh3.googleusercontent.com/-cQLcFi7r2uc/AAAAAAAAAAI/AAAAAAAAAAA/ACHi3rfoCSVbR8qVruV55uAYdSC-znVn2w.CMID/s96-c/photo.jpg",
  first_name: "Phillip",
  last_name: "McPhillipson",
  title: "Founder",
  phone_country_code: "",
  phone: "",
  phone_ext: "",
  company: { label: "Glacier Associates", value: "123456789" },
  company_roles: ["owner", "asset_manager", "property_manager"],
  office_street: "1730 Minor Avenue",
  office_city: "Lansing",
  office_state: { label: "Michigan", value: "Michigan" },
  office_zip: "11111",
  office_country: { label: "United States of America", value: "USA" },
  office_address: "1730 Minor Avenue, Lansing, MI",
  office_name: "Michigan",
  office_type: 3
};

const tabsOrder = ["portfolio", "group", "property"];

const initialTab = "portfolio";

export const props = {
  rules,
  profile,
  user,
  company_roles,
  office_options,
  itemsOrder,
  tabsOrder,
  initialTab,
  office_countries,
  us_state_list,
  gb_county_list,
  dispatch: () => {}
};
