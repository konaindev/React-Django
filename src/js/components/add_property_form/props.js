export const props = {
  post_url: "/projects/new",
  states: [
    { id: "AL", name: "AL" },
    { id: "GA", name: "GA" },
    { id: "NE", name: "NE" },
    { id: "OR", name: "OR" },
    { id: "WY", name: "WY" }
  ],
  packages: [
    { id: "accelerate", name: "Accelerate" },
    { id: "optimize", name: "Optimize" },
    { id: "ground", name: "Ground Up" },
    { id: "other", name: "Not Sure" }
  ]
};
