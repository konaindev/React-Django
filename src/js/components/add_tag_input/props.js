const suggestedTags = [
  { name: "Tag 1", count: 5 },
  { name: "Test Tag 2", count: 3 },
  { name: "Lake Highland Properties 3", count: 2 },
  { name: "New Tag", count: 1 }
];

export default { suggestedTags };
