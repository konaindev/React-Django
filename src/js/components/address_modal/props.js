export default {
  isOpen: true,
  addresses: {
    suggested_address: {
      office_city: "Washington",
      office_state: "DC",
      office_street: "1600 Pennsylvania Ave NW",
      office_zip: "20500",
      office_country: "United States"
    }
  },
  dispatch_type: "FAKE DISPATCH"
};
