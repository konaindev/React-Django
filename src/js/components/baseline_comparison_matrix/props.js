import {
  baselineReport,
  baselineReportWithoutCompetitors,
  baselineReportWithOneCompetitor
} from "../common_report/props";

export const props = { report: baselineReport };
export const no_competitor_props = { report: baselineReportWithoutCompetitors };
export const one_competitor_props = {
  report: baselineReportWithOneCompetitor
};
