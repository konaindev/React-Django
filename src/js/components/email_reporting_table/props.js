const properties = [
  { id: 1, name: "Adams Lane Flats" },
  { id: 2, name: "Appleton Farms" },
  { id: 3, name: "Cue Townhomes" },
  { id: 4, name: "Over the Rhine" },
  { id: 5, name: "Rainier Lofts" },
  { id: 6, name: "Sea Breeze Lofts" },
  { id: 7, name: "Sunrise Lofts" },
  { id: 8, name: "Tamarand Lofts" },
  { id: 9, name: "Viscoe Apartments" },
  { id: 10, name: "Vue Apartments" }
];

const groups = [
  { id: 1, name: "Southwestern", propertiesCount: 6 },
  { id: 2, name: "Tri-City", propertiesCount: 5 }
];

const portfolio = [
  { id: 1, name: "Portfolio", groupsCount: 2, propertiesCount: 16 }
];

export { properties, groups, portfolio };
