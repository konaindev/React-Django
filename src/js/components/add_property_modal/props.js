import { states } from "../add_property_form/states";

export const props = {
  formProps: {
    post_url: null,
    packages: [
      { id: "accelerate", name: "Accelerate" },
      { id: "optimize", name: "Optimize" },
      { id: "ground", name: "Ground Up" },
      { id: "other", name: "Not Sure" }
    ],
    states
  }
};
