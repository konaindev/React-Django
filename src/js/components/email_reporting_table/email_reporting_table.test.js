import React from "react";
import renderer from "react-test-renderer";

import EmailReportingTable from "./index";
import { groups, portfolio, properties } from "./props";

describe("EmailReportingTable", () => {
  it("renders portfolio table", () => {
    const tree = renderer
      .create(
        <EmailReportingTable
          properties={portfolio}
          propertiesCount={portfolio.length}
        />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
  it("renders groups table", () => {
    const tree = renderer
      .create(
        <EmailReportingTable
          properties={groups}
          propertiesCount={groups.length}
        />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
  it("renders properties table", () => {
    const tree = renderer
      .create(
        <EmailReportingTable
          properties={properties}
          propertiesCount={properties.length}
        />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
  it("has `view more` button", () => {
    const tree = renderer
      .create(
        <EmailReportingTable
          properties={properties.slice(0, 5)}
          showLoadBtn={true}
        />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
