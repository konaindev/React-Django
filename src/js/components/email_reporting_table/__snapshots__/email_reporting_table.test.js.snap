// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmailReportingTable has \`view more\` button 1`] = `
<div
  className="email-reporting-table"
>
  <div
    className="email-reporting-table__rows"
  >
    <div
      className="email-reporting-row email-reporting-table__row"
    >
      <div
        className="email-reporting-row__col"
      >
        <div
          className="email-reporting-row__title"
        >
          Adams Lane Flats
        </div>
      </div>
      <div
        className="email-reporting-row__col email-reporting-row__col--middle"
      >
        <label
          className="button-toggle button-toggle--unchecked email-reporting-row__toggle"
        >
          <input
            checked={false}
            className="button-toggle__input"
            onChange={[Function]}
            type="checkbox"
          />
          <span
            className="button-toggle__track"
          >
            <span
              className="button-toggle__thumb"
            />
            <div
              className="button-toggle__checked"
            >
              <div
                className="button-toggle__inner-label"
              >
                On
              </div>
            </div>
            <div
              className="button-toggle__unchecked"
            >
              <div
                className="button-toggle__inner-label"
              >
                Off
              </div>
            </div>
          </span>
          
        </label>
      </div>
    </div>
    <div
      className="email-reporting-row email-reporting-table__row"
    >
      <div
        className="email-reporting-row__col"
      >
        <div
          className="email-reporting-row__title"
        >
          Appleton Farms
        </div>
      </div>
      <div
        className="email-reporting-row__col email-reporting-row__col--middle"
      >
        <label
          className="button-toggle button-toggle--unchecked email-reporting-row__toggle"
        >
          <input
            checked={false}
            className="button-toggle__input"
            onChange={[Function]}
            type="checkbox"
          />
          <span
            className="button-toggle__track"
          >
            <span
              className="button-toggle__thumb"
            />
            <div
              className="button-toggle__checked"
            >
              <div
                className="button-toggle__inner-label"
              >
                On
              </div>
            </div>
            <div
              className="button-toggle__unchecked"
            >
              <div
                className="button-toggle__inner-label"
              >
                Off
              </div>
            </div>
          </span>
          
        </label>
      </div>
    </div>
    <div
      className="email-reporting-row email-reporting-table__row"
    >
      <div
        className="email-reporting-row__col"
      >
        <div
          className="email-reporting-row__title"
        >
          Cue Townhomes
        </div>
      </div>
      <div
        className="email-reporting-row__col email-reporting-row__col--middle"
      >
        <label
          className="button-toggle button-toggle--unchecked email-reporting-row__toggle"
        >
          <input
            checked={false}
            className="button-toggle__input"
            onChange={[Function]}
            type="checkbox"
          />
          <span
            className="button-toggle__track"
          >
            <span
              className="button-toggle__thumb"
            />
            <div
              className="button-toggle__checked"
            >
              <div
                className="button-toggle__inner-label"
              >
                On
              </div>
            </div>
            <div
              className="button-toggle__unchecked"
            >
              <div
                className="button-toggle__inner-label"
              >
                Off
              </div>
            </div>
          </span>
          
        </label>
      </div>
    </div>
    <div
      className="email-reporting-row email-reporting-table__row"
    >
      <div
        className="email-reporting-row__col"
      >
        <div
          className="email-reporting-row__title"
        >
          Over the Rhine
        </div>
      </div>
      <div
        className="email-reporting-row__col email-reporting-row__col--middle"
      >
        <label
          className="button-toggle button-toggle--unchecked email-reporting-row__toggle"
        >
          <input
            checked={false}
            className="button-toggle__input"
            onChange={[Function]}
            type="checkbox"
          />
          <span
            className="button-toggle__track"
          >
            <span
              className="button-toggle__thumb"
            />
            <div
              className="button-toggle__checked"
            >
              <div
                className="button-toggle__inner-label"
              >
                On
              </div>
            </div>
            <div
              className="button-toggle__unchecked"
            >
              <div
                className="button-toggle__inner-label"
              >
                Off
              </div>
            </div>
          </span>
          
        </label>
      </div>
    </div>
    <div
      className="email-reporting-row email-reporting-table__row"
    >
      <div
        className="email-reporting-row__col"
      >
        <div
          className="email-reporting-row__title"
        >
          Rainier Lofts
        </div>
      </div>
      <div
        className="email-reporting-row__col email-reporting-row__col--middle"
      >
        <label
          className="button-toggle button-toggle--unchecked email-reporting-row__toggle"
        >
          <input
            checked={false}
            className="button-toggle__input"
            onChange={[Function]}
            type="checkbox"
          />
          <span
            className="button-toggle__track"
          >
            <span
              className="button-toggle__thumb"
            />
            <div
              className="button-toggle__checked"
            >
              <div
                className="button-toggle__inner-label"
              >
                On
              </div>
            </div>
            <div
              className="button-toggle__unchecked"
            >
              <div
                className="button-toggle__inner-label"
              >
                Off
              </div>
            </div>
          </span>
          
        </label>
      </div>
    </div>
  </div>
  <div
    className="email-reporting-table__loading"
  >
    <button
      className="button button--secondary email-reporting-table__loading-btn"
      onClick={[Function]}
    >
      View More
    </button>
  </div>
</div>
`;

exports[`EmailReportingTable renders groups table 1`] = `
<div
  className="email-reporting-table"
>
  <div
    className="email-reporting-table__rows"
  >
    <div
      className="email-reporting-row email-reporting-table__row"
    >
      <div
        className="email-reporting-row__col"
      >
        <div
          className="email-reporting-row__title"
        >
          Southwestern
        </div>
        <div
          className="email-reporting-row__info"
        >
          6 Properties
        </div>
      </div>
      <div
        className="email-reporting-row__col email-reporting-row__col--middle"
      >
        <label
          className="button-toggle button-toggle--unchecked email-reporting-row__toggle"
        >
          <input
            checked={false}
            className="button-toggle__input"
            onChange={[Function]}
            type="checkbox"
          />
          <span
            className="button-toggle__track"
          >
            <span
              className="button-toggle__thumb"
            />
            <div
              className="button-toggle__checked"
            >
              <div
                className="button-toggle__inner-label"
              >
                On
              </div>
            </div>
            <div
              className="button-toggle__unchecked"
            >
              <div
                className="button-toggle__inner-label"
              >
                Off
              </div>
            </div>
          </span>
          
        </label>
      </div>
    </div>
    <div
      className="email-reporting-row email-reporting-table__row"
    >
      <div
        className="email-reporting-row__col"
      >
        <div
          className="email-reporting-row__title"
        >
          Tri-City
        </div>
        <div
          className="email-reporting-row__info"
        >
          5 Properties
        </div>
      </div>
      <div
        className="email-reporting-row__col email-reporting-row__col--middle"
      >
        <label
          className="button-toggle button-toggle--unchecked email-reporting-row__toggle"
        >
          <input
            checked={false}
            className="button-toggle__input"
            onChange={[Function]}
            type="checkbox"
          />
          <span
            className="button-toggle__track"
          >
            <span
              className="button-toggle__thumb"
            />
            <div
              className="button-toggle__checked"
            >
              <div
                className="button-toggle__inner-label"
              >
                On
              </div>
            </div>
            <div
              className="button-toggle__unchecked"
            >
              <div
                className="button-toggle__inner-label"
              >
                Off
              </div>
            </div>
          </span>
          
        </label>
      </div>
    </div>
  </div>
</div>
`;

exports[`EmailReportingTable renders portfolio table 1`] = `
<div
  className="email-reporting-table"
>
  <div
    className="email-reporting-table__rows"
  >
    <div
      className="email-reporting-row email-reporting-table__row"
    >
      <div
        className="email-reporting-row__col"
      >
        <div
          className="email-reporting-row__title"
        >
          Portfolio
        </div>
        <div
          className="email-reporting-row__info"
        >
          2 Groups, 16 Properties
        </div>
      </div>
      <div
        className="email-reporting-row__col email-reporting-row__col--middle"
      >
        <label
          className="button-toggle button-toggle--unchecked email-reporting-row__toggle"
        >
          <input
            checked={false}
            className="button-toggle__input"
            onChange={[Function]}
            type="checkbox"
          />
          <span
            className="button-toggle__track"
          >
            <span
              className="button-toggle__thumb"
            />
            <div
              className="button-toggle__checked"
            >
              <div
                className="button-toggle__inner-label"
              >
                On
              </div>
            </div>
            <div
              className="button-toggle__unchecked"
            >
              <div
                className="button-toggle__inner-label"
              >
                Off
              </div>
            </div>
          </span>
          
        </label>
      </div>
    </div>
  </div>
</div>
`;

exports[`EmailReportingTable renders properties table 1`] = `
<div
  className="email-reporting-table"
>
  <div
    className="email-reporting-table__rows"
  >
    <div
      className="email-reporting-row email-reporting-table__row"
    >
      <div
        className="email-reporting-row__col"
      >
        <div
          className="email-reporting-row__title"
        >
          Adams Lane Flats
        </div>
      </div>
      <div
        className="email-reporting-row__col email-reporting-row__col--middle"
      >
        <label
          className="button-toggle button-toggle--unchecked email-reporting-row__toggle"
        >
          <input
            checked={false}
            className="button-toggle__input"
            onChange={[Function]}
            type="checkbox"
          />
          <span
            className="button-toggle__track"
          >
            <span
              className="button-toggle__thumb"
            />
            <div
              className="button-toggle__checked"
            >
              <div
                className="button-toggle__inner-label"
              >
                On
              </div>
            </div>
            <div
              className="button-toggle__unchecked"
            >
              <div
                className="button-toggle__inner-label"
              >
                Off
              </div>
            </div>
          </span>
          
        </label>
      </div>
    </div>
    <div
      className="email-reporting-row email-reporting-table__row"
    >
      <div
        className="email-reporting-row__col"
      >
        <div
          className="email-reporting-row__title"
        >
          Appleton Farms
        </div>
      </div>
      <div
        className="email-reporting-row__col email-reporting-row__col--middle"
      >
        <label
          className="button-toggle button-toggle--unchecked email-reporting-row__toggle"
        >
          <input
            checked={false}
            className="button-toggle__input"
            onChange={[Function]}
            type="checkbox"
          />
          <span
            className="button-toggle__track"
          >
            <span
              className="button-toggle__thumb"
            />
            <div
              className="button-toggle__checked"
            >
              <div
                className="button-toggle__inner-label"
              >
                On
              </div>
            </div>
            <div
              className="button-toggle__unchecked"
            >
              <div
                className="button-toggle__inner-label"
              >
                Off
              </div>
            </div>
          </span>
          
        </label>
      </div>
    </div>
    <div
      className="email-reporting-row email-reporting-table__row"
    >
      <div
        className="email-reporting-row__col"
      >
        <div
          className="email-reporting-row__title"
        >
          Cue Townhomes
        </div>
      </div>
      <div
        className="email-reporting-row__col email-reporting-row__col--middle"
      >
        <label
          className="button-toggle button-toggle--unchecked email-reporting-row__toggle"
        >
          <input
            checked={false}
            className="button-toggle__input"
            onChange={[Function]}
            type="checkbox"
          />
          <span
            className="button-toggle__track"
          >
            <span
              className="button-toggle__thumb"
            />
            <div
              className="button-toggle__checked"
            >
              <div
                className="button-toggle__inner-label"
              >
                On
              </div>
            </div>
            <div
              className="button-toggle__unchecked"
            >
              <div
                className="button-toggle__inner-label"
              >
                Off
              </div>
            </div>
          </span>
          
        </label>
      </div>
    </div>
    <div
      className="email-reporting-row email-reporting-table__row"
    >
      <div
        className="email-reporting-row__col"
      >
        <div
          className="email-reporting-row__title"
        >
          Over the Rhine
        </div>
      </div>
      <div
        className="email-reporting-row__col email-reporting-row__col--middle"
      >
        <label
          className="button-toggle button-toggle--unchecked email-reporting-row__toggle"
        >
          <input
            checked={false}
            className="button-toggle__input"
            onChange={[Function]}
            type="checkbox"
          />
          <span
            className="button-toggle__track"
          >
            <span
              className="button-toggle__thumb"
            />
            <div
              className="button-toggle__checked"
            >
              <div
                className="button-toggle__inner-label"
              >
                On
              </div>
            </div>
            <div
              className="button-toggle__unchecked"
            >
              <div
                className="button-toggle__inner-label"
              >
                Off
              </div>
            </div>
          </span>
          
        </label>
      </div>
    </div>
    <div
      className="email-reporting-row email-reporting-table__row"
    >
      <div
        className="email-reporting-row__col"
      >
        <div
          className="email-reporting-row__title"
        >
          Rainier Lofts
        </div>
      </div>
      <div
        className="email-reporting-row__col email-reporting-row__col--middle"
      >
        <label
          className="button-toggle button-toggle--unchecked email-reporting-row__toggle"
        >
          <input
            checked={false}
            className="button-toggle__input"
            onChange={[Function]}
            type="checkbox"
          />
          <span
            className="button-toggle__track"
          >
            <span
              className="button-toggle__thumb"
            />
            <div
              className="button-toggle__checked"
            >
              <div
                className="button-toggle__inner-label"
              >
                On
              </div>
            </div>
            <div
              className="button-toggle__unchecked"
            >
              <div
                className="button-toggle__inner-label"
              >
                Off
              </div>
            </div>
          </span>
          
        </label>
      </div>
    </div>
    <div
      className="email-reporting-row email-reporting-table__row"
    >
      <div
        className="email-reporting-row__col"
      >
        <div
          className="email-reporting-row__title"
        >
          Sea Breeze Lofts
        </div>
      </div>
      <div
        className="email-reporting-row__col email-reporting-row__col--middle"
      >
        <label
          className="button-toggle button-toggle--unchecked email-reporting-row__toggle"
        >
          <input
            checked={false}
            className="button-toggle__input"
            onChange={[Function]}
            type="checkbox"
          />
          <span
            className="button-toggle__track"
          >
            <span
              className="button-toggle__thumb"
            />
            <div
              className="button-toggle__checked"
            >
              <div
                className="button-toggle__inner-label"
              >
                On
              </div>
            </div>
            <div
              className="button-toggle__unchecked"
            >
              <div
                className="button-toggle__inner-label"
              >
                Off
              </div>
            </div>
          </span>
          
        </label>
      </div>
    </div>
    <div
      className="email-reporting-row email-reporting-table__row"
    >
      <div
        className="email-reporting-row__col"
      >
        <div
          className="email-reporting-row__title"
        >
          Sunrise Lofts
        </div>
      </div>
      <div
        className="email-reporting-row__col email-reporting-row__col--middle"
      >
        <label
          className="button-toggle button-toggle--unchecked email-reporting-row__toggle"
        >
          <input
            checked={false}
            className="button-toggle__input"
            onChange={[Function]}
            type="checkbox"
          />
          <span
            className="button-toggle__track"
          >
            <span
              className="button-toggle__thumb"
            />
            <div
              className="button-toggle__checked"
            >
              <div
                className="button-toggle__inner-label"
              >
                On
              </div>
            </div>
            <div
              className="button-toggle__unchecked"
            >
              <div
                className="button-toggle__inner-label"
              >
                Off
              </div>
            </div>
          </span>
          
        </label>
      </div>
    </div>
    <div
      className="email-reporting-row email-reporting-table__row"
    >
      <div
        className="email-reporting-row__col"
      >
        <div
          className="email-reporting-row__title"
        >
          Tamarand Lofts
        </div>
      </div>
      <div
        className="email-reporting-row__col email-reporting-row__col--middle"
      >
        <label
          className="button-toggle button-toggle--unchecked email-reporting-row__toggle"
        >
          <input
            checked={false}
            className="button-toggle__input"
            onChange={[Function]}
            type="checkbox"
          />
          <span
            className="button-toggle__track"
          >
            <span
              className="button-toggle__thumb"
            />
            <div
              className="button-toggle__checked"
            >
              <div
                className="button-toggle__inner-label"
              >
                On
              </div>
            </div>
            <div
              className="button-toggle__unchecked"
            >
              <div
                className="button-toggle__inner-label"
              >
                Off
              </div>
            </div>
          </span>
          
        </label>
      </div>
    </div>
    <div
      className="email-reporting-row email-reporting-table__row"
    >
      <div
        className="email-reporting-row__col"
      >
        <div
          className="email-reporting-row__title"
        >
          Viscoe Apartments
        </div>
      </div>
      <div
        className="email-reporting-row__col email-reporting-row__col--middle"
      >
        <label
          className="button-toggle button-toggle--unchecked email-reporting-row__toggle"
        >
          <input
            checked={false}
            className="button-toggle__input"
            onChange={[Function]}
            type="checkbox"
          />
          <span
            className="button-toggle__track"
          >
            <span
              className="button-toggle__thumb"
            />
            <div
              className="button-toggle__checked"
            >
              <div
                className="button-toggle__inner-label"
              >
                On
              </div>
            </div>
            <div
              className="button-toggle__unchecked"
            >
              <div
                className="button-toggle__inner-label"
              >
                Off
              </div>
            </div>
          </span>
          
        </label>
      </div>
    </div>
    <div
      className="email-reporting-row email-reporting-table__row"
    >
      <div
        className="email-reporting-row__col"
      >
        <div
          className="email-reporting-row__title"
        >
          Vue Apartments
        </div>
      </div>
      <div
        className="email-reporting-row__col email-reporting-row__col--middle"
      >
        <label
          className="button-toggle button-toggle--unchecked email-reporting-row__toggle"
        >
          <input
            checked={false}
            className="button-toggle__input"
            onChange={[Function]}
            type="checkbox"
          />
          <span
            className="button-toggle__track"
          >
            <span
              className="button-toggle__thumb"
            />
            <div
              className="button-toggle__checked"
            >
              <div
                className="button-toggle__inner-label"
              >
                On
              </div>
            </div>
            <div
              className="button-toggle__unchecked"
            >
              <div
                className="button-toggle__inner-label"
              >
                Off
              </div>
            </div>
          </span>
          
        </label>
      </div>
    </div>
  </div>
</div>
`;
